cibuildwheel/util.py

```diff
@@ -93,7 +93,24 @@ def build_frontend_or_default(
     os.environ.get("CIBW_CACHE_PATH", DEFAULT_CIBW_CACHE_PATH)
 ).resolve()
 
-IS_WIN: Final[bool] = sys.platform.startswith("win")
+
+def get_host_platform() -> PlatformName | None:
+    # Cross-platform support. Used for --print-build-identifiers or docker builds.
+    if sys.platform.startswith("linux"):
+        return "linux"
+    elif sys.platform.startswith("darwin"):
+        return "macos"
+    elif sys.platform.startswith("win"):
+        return "windows"
+    else:
+        return None
+
+
+HOST_PLATFORM: Final[PlatformName | None] = get_host_platform()
+
+HOST_IS_WINDOWS: Final[bool] = get_host_platform() == "windows"
+
+HOST_IS_WIN32: Final[bool] = sys.platform == "win32"
```

Comment:
This might be a bit excessive but I think it makes the intended distinction between these two checks 100% clear.